participants/Fouesneau/pr_tutorial/simple_functions.py

```diff
@@ -1,11 +1,15 @@
-def fibonacci(max):
+from typing import Sequence
+
+def fibonacci(max: int) -> List[int]:
+    """ Calculate the Fibonacci series up to max """
     values = [0, 1]
     while values[-2] + values[-1] < max:
         values.append(values[-2] + values[-1])
     return values
 
 
-def factorial(value):
+def factorial(value: int) -> int:
```

Comment:
Just realized that the function will only work if the value is positive. Maybe it'd be great to add this in the docstring?

Reply:
And also would it be worth preventing the user with a positional argument like `factorial(value=something)`? We could use the following
```suggestion
def factorial(value: int, /) -> int:
```

Reply:
Thanks, but that's a style change. The example you mention will raise an error already

Reply:
On my machine, it seems to be working ok
```python
In [4]: factorial(value=10)
Out[4]: 3628800
```
But that's indeed purely stylistic :)